build.md

```diff
@@ -1,59 +1,55 @@
 # The Compose Specification - Build support
 {:.no_toc}
 
-*Note:* Build is an OPTIONAL part of the Compose Specification
+*Note:* Build is an OPTIONAL part of the Compose specification.
 
 * ToC
 {:toc}
 
 ## Introduction
 
-Compose specification is a platform-neutral way to define multi-container applications. A Compose implementation
-focusing on development use-case to run application on local machine will obviously also support (re)building
-application from sources. The Compose Build specification allows to define the build process within a Compose file
+The Compose specification is a platform-neutral way to define multi-container applications. A Compose implementation which focuses on running an application on a local machine will obviously need to alo support (re)building
+application from source. The Compose Build specification lets developers define the build process within a Compose file
 in a portable way.
 
 ## Definitions
 
-Compose Specification is extended to support an OPTIONAL `build` subsection on services. This section define the
-build requirements for service container image. Only a subset of Compose file services MAY define such a Build
-subsection, others being created based on `Image` attribute. When a Build subsection is present for a service, it
-is *valid* for a Compose file to miss an `Image` attribute for corresponding service, as Compose implementation
-can build image from source.
+Compose Specification can be extended to support an OPTIONAL `build` subsection on services. 
+
+This section defines the build requirements for a service container image. Only a subset of Compose file services MAY define such a `build`
```

Comment:
This sort of reads like not ALL services in a Compose file MAY define a build section, while what it's trying to say is that not all services MUST define a build section as far as I can tell. Maybe we can reword this?